docker-compose-examples/keycloak-mssql.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.2"
+
+services:
+    mssql:
+        image: mcr.microsoft.com/mssql/server
+        volumes:
+            - type: volume
+              source: keycloakdbvolume
+              target: /var/opt/mssql
+        ports:
+            - "1433:1433"
+        environment:
+            - ACCEPT_EULA=Y
+            - SA_PASSWORD=Password!23
+            - MSSQL_PID=Developer
+    mssqlscripts:
+        image: mcr.microsoft.com/mssql-tools
+        depends_on:
+            - mssql
+        command: /bin/bash -c 'until /opt/mssql-tools/bin/sqlcmd -S mssql -U sa -P "Password!23" -Q "create database Keycloak"; do sleep 5; done'
+    keycloak:
+        image: local/keycloak-rolandta
```

Comment:
Shouldn't use jboss/keycloak image instead of local/keycloak-rolandta ?

Reply:
Yes, good catch - that was my local tag when I was building the original fork.